src/pages/index.tsx

```diff
@@ -0,0 +1,110 @@
+import React, { useCallback, useState } from "react";
+import Link from "@docusaurus/Link";
+import useDocusaurusContext from "@docusaurus/useDocusaurusContext";
+import { CopyIcon, GithubIcon } from "../components/Icons";
+
+import { ThemeConfig } from "@docusaurus/preset-classic";
+import Features from "../components/Features";
+
+function copyToClipboard(text: string) {
+  if (navigator.clipboard) {
+    navigator.clipboard.writeText(text);
+    return;
+  }
+
+  var textArea = document.createElement("textarea");
+  textArea.value = text;
+
+  textArea.style.top = "0";
+  textArea.style.left = "0";
+  textArea.style.position = "fixed";
+
+  document.body.appendChild(textArea);
+  textArea.focus();
+  textArea.select();
+
+  try {
+    document.execCommand("copy");
+  } catch (err) {}
+
+  document.body.removeChild(textArea);
+}
+
+export default function Home() {
+  const {
+    siteConfig: { themeConfig, tagline },
+  } = useDocusaurusContext();
+  const headerImg = (themeConfig as ThemeConfig).navbar!.logo!;
+
+  const [copied, setCopied] = useState(false);
+
+  const copyCommandHandler = useCallback(() => {
+    if (copied) return;
+
+    copyToClipboard(
+      "dart pub global activate stacked_cli\nstacked create app awesome_app"
+    );
+    setCopied(true);
+
+    setTimeout(() => setCopied(false), 3000);
+  }, []);
+
+  return (
+    <>
+      <header className="sticky top-0 flex items-center justify-between px-6 py-4 shadow bg-white/20 backdrop-blur-md">
```

Comment:
Could we remove the header from the landing page. I'd like getStarted to be the only option for the user, the rest of the docs can have the header with the Stacked icon and the github icon.